src/dmd/root/outbuffer.d

```diff
@@ -323,7 +326,7 @@ struct OutBuffer
                 va_end(va);
                 if (count == -1)
                     psize *= 2;
-                else if (count >= psize)
+                else if (!DMCRT && count >= psize)
```

Comment:
Do you mean the `count` is wrong for DM's `vsnprintf`? If so, please have a test case? Bugzilla issue?